packages/caip/src/index.ts

```diff
@@ -1,10 +1,6 @@
 import * as adapters from './adapters'
```

Comment:
Can reduce import + export to `export * as adapters from './adapters'`

Reply:
Addressed in https://github.com/shapeshift/lib/pull/625